aspnetcore/toc.md

```diff
@@ -1,223 +1,201 @@
-#### [Add a model](xref:tutorials/razor-pages-vsc/model)
-#### [Scaffolded Razor Pages](xref:tutorials/razor-pages-vsc/page)
-#### [Work with SQLite](xref:tutorials/razor-pages-vsc/sql)
-#### [Update the pages](xref:tutorials/razor-pages-vsc/da1)
-#### [Add search](xref:tutorials/razor-pages-vsc/search)
-
-### [MVC web app with Visual Studio for Mac](xref:tutorials/first-mvc-app-mac/index)
-#### [Get started](xref:tutorials/first-mvc-app-mac/start-mvc)
-#### [Add a controller](xref:tutorials/first-mvc-app-mac/adding-controller)
-#### [Add a view](xref:tutorials/first-mvc-app-mac/adding-view)
-#### [Add a model](xref:tutorials/first-mvc-app-mac/adding-model)
-#### [Work with SQLite](xref:tutorials/first-mvc-app-mac/working-with-sql)
-#### [Controller methods and views](xref:tutorials/first-mvc-app-mac/controller-methods-views)
-#### [Add search](xref:tutorials/first-mvc-app-mac/search)
-#### [Add a new field](xref:tutorials/first-mvc-app-mac/new-field)
-#### [Add validation](xref:tutorials/first-mvc-app-mac/validation)
-#### [Examine the Details and Delete methods](xref:tutorials/first-mvc-app/details)
-
-### [MVC web app with Visual Studio Code on macOS or Linux](xref:tutorials/first-mvc-app-xplat/index)
-#### [Get started](tutorials/first-mvc-app-xplat/start-mvc.md)
-#### [Add a controller](tutorials/first-mvc-app-xplat/adding-controller.md)
-#### [Add a view](tutorials/first-mvc-app-xplat/adding-view.md)
-#### [Add a model](tutorials/first-mvc-app-xplat/adding-model.md)
-#### [Work with SQLite](tutorials/first-mvc-app-xplat/working-with-sql.md)
-#### [Controller methods and views](tutorials/first-mvc-app-xplat/controller-methods-views.md)
-#### [Add search](tutorials/first-mvc-app-xplat/search.md)
-#### [Add a new field](tutorials/first-mvc-app-xplat/new-field.md)
-#### [Add validation](tutorials/first-mvc-app-xplat/validation.md)
-#### [Examine the Details and Delete methods](tutorials/first-mvc-app/details.md)
-
-### [Web API with Visual Studio for Mac](xref:tutorials/first-web-api-mac)
-### [Web API with Visual Studio Code](xref:tutorials/web-api-vsc)
-
-## [Develop apps using a file watcher](xref:tutorials/dotnet-watch)
-
-## [Create backend services for mobile apps](mobile/native-mobile-backend.md)
-
-# [Fundamentals](xref:fundamentals/index)
-## [Application startup](xref:fundamentals/startup)
+# Fundamentals
+## [Overview](xref:fundamentals/index)
+## [App startup](xref:fundamentals/startup)
 ## [Dependency injection (services)](xref:fundamentals/dependency-injection)
-## [Middleware](xref:fundamentals/middleware/index)
+## [Routing](xref:fundamentals/routing)
+## [Multiple environments](xref:fundamentals/environments)
+## [Configuration](xref:fundamentals/configuration/index)
+## [Options](xref:fundamentals/configuration/options)
+## [Logging](xref:fundamentals/logging/index)
+## [Handle errors](xref:fundamentals/error-handling)
+## Middleware
 ### [Middleware](xref:fundamentals/middleware/index)
```

Comment:
Should we go with "Overview" for this link's text?

Reply:
Or should the other two go to Other topics?